src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3743,6 +3745,21 @@ def update_vmss(cmd, resource_group_name, name, license_type=None, no_wait=False
             'vTpmEnabled': enable_vtpm
         }}
 
+    if regular_priority_count is not None or regular_priority_percentage is not None:
+        if vmss.orchestration_mode != 'Flexible':
```

Comment:
> if namespace.orchestration_mode.lower() != flexible_str.lower():

Why not keep consistent with the above verification logic?

Reply:
From service error: Specify priorityMixPolicy is allowed only for virtual machine scale sets with orchestration mode 'flexible'

Reply:
I mean, check in lowercase